employees.html

```diff
@@ -0,0 +1,559 @@
+              <hr class="all-dashboard-heading-line" />
+              <p>ALL</p>
+            </div>
+            <ul>
+              <li class="roles-selection-menu">
+                <a href="#">
+                  <div>
+                    <img
+                      src="images/horizontal-nav/Dashboard.svg"
+                      alt="Access-rights"
+                    />
+                    Dashboard
+                  </div>
+                  <img
+                    src="images/drop-right-icon.png"
+                    alt="drop-right"
+                    class="icon-none"
+                /></a>
+              </li>
+              <li class="roles-selected-menu">
+                <a href="#">
+                  <div>
+                    <img
+                      src="images/horizontal-nav/Employees.svg"
+                      alt="employess"
+                    />
+                    Employees
+                  </div>
+                </a>
+              </li>
+              <span class="dashboard-selection-highlight"></span>
+            </ul>
+          </div>
+          <div class="role-management-dashboard">
+            <div class="role-management-dashboard-heading">
+              <hr class="role-management-dashboard-heading-line" />
+              <p>ROLE / USER MANAGEMENT</p>
+            </div>
+            <ul>
+              <li class="roles-selection-menu">
+                <a href="roles.html">
+                  <div>
+                    <img
+                      src="images/horizontal-nav/Roles.svg"
+                      alt="Access-rights"
+                    />
+                    Roles
+                  </div>
+                  <img
+                    src="images/drop-right-icon.png"
+                    alt="drop-right"
+                    class="roles-selection-menu-icon"
+                /></a>
+              </li>
+
+              <li class="roles-selection-menu">
+                <a href="#">
+                  <div>
+                    <img
+                      src="images\horizontal-nav\assign user.svg"
+                      alt="Access-rights"
+                    />
+                    Access Rights
+                  </div>
+                  <img
+                    src="images/drop-right-icon.png"
+                    alt="drop-right"
+                    class="roles-selection-menu-icon"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="app-update">
+          <p class="app-update-install">Install new app-update 2.3.1A</p>
+          <p class="app-update-description">
+            Allow to launch the new app-update for the AD application
+          </p>
+          <div>
+            <button class="btn btn-dismiss">Dismiss</button
+            ><button class="btn btn-accept">Accept</button>
+          </div>
+        </div>
+      </aside>
+      <header class="header-menu">
+        <div class="navbar">

```

Comment:
change classname, don't use navbar, there are no navigation links inside
use meaning full classnames

Reply:
section containing the navbar was renamed but there was no better name for the container as it also contains the options to change user profile and for that there can also be a option to sign out which would be a navigational link again. but still im chagning the name od this container too.